neurolib/models/wc/timeIntegration.py

```diff
@@ -72,44 +72,44 @@ def timeIntegration(params):
     max_global_delay = np.max(Dmat_ndt)
     startind = int(max_global_delay + 1)  # timestep to start integration at
 
-    x_ou = params["x_ou"]
-    y_ou = params["y_ou"]
+    e_ou = params["e_ou"]
+    i_ou = params["i_ou"]
 
-    x_ext = params["x_ext"]
-    y_ext = params["y_ext"]
+    e_ext = params["e_ext"]
+    i_ext = params["i_ext"]
 
-    # state variable arrays, have length of t + startind
-    # they store initial conditions AND simulated data
-    xs = np.zeros((N, startind + len(t)))
-    ys = np.zeros((N, startind + len(t)))
+
+    # set of the state variable arrays
+    exc = np.zeros((N, len(t)))
+    inh = np.zeros((N, len(t)))
 
     # ------------------------------------------------------------------------
     # Set initial values
     # if initial values are just a Nx1 array
-    if np.shape(params["xs_init"])[1] == 1:
-        xs_init = np.dot(params["xs_init"], np.ones((1, startind)))
-        ys_init = np.dot(params["ys_init"], np.ones((1, startind)))
+    if np.shape(params["es_init"])[1] == 1:
+        es_init = np.dot(params["es_init"], np.ones((1, startind)))
+        is_init = np.dot(params["is_init"], np.ones((1, startind)))
```

Comment:
ideally, these should be called `exc_init` and `inh_init`
```suggestion
        exc_init = np.dot(params["ecx_init"], np.ones((1, startind)))
        inh_init = np.dot(params["inh_init"], np.ones((1, startind)))
```